src/gui/optionsdialog.ui

```diff
@@ -769,9 +769,9 @@
                 </widget>
                </item>
                <item>
-                <widget class="QCheckBox" name="checkCreateSubfolder">
+                <widget class="QCheckBox" name="checkKeepSubfolder">
                  <property name="text">
-                  <string>Create subfolder for torrents with multiple files</string>
+                  <string>Keep subfolder for torrents with multiple files</string>
```

Comment:
What's the deal with "with multiple files"? It'll still create the folder if there's only 1 file (with root folder).
Should i change/remove it: "Keep the top-level folder" or omit "the" to keep it consistent with the option in other places?

Reply:
@thalieht 
> omit "the"

Probably a good idea. 

>What's the deal with "with multiple files"?

That part should probably be deleted.

Reply:
>It'll still create the folder if there's only 1 file (with root folder).

Yes. But qBittorrent doesn't apply this option to torrent with single file, AFAIK.

Reply:
>Yes. But qBittorrent doesn't apply this option to torrent with single file, AFAIK.

You mean 1 file with a root folder or just single file? Because in the former case it removes the root folder just fine and in the latter case, this option has no effect on torrents without root folder, no matter the file count AFAIK (can't find a torrent with multiple files + no root folder and i can't figure out how to make one with qBt torrent creator).